### ISSUE TITLE
Cablescan plugin doesn't work with SAT>IP

### ISSUE THREAD

**Issue by retrofan1979** [OpenViX/enigma2#571]
The cablescan plugin doesn't work with SAT>IP server as VTUNER. I got this error: Scanning failed! I think it's because my SAT>IP server wants the (Spectrum) Inversion setting to be turned on and indeed I don't see this setting. Can you please add this option to the cablescan plugin?
Can you please compile an IPK, so I can use it right away on all compatible images besides OpenVIX? Thank you!

Here's the source code of the cablescan plugin:

https://github.com/OpenViX/enigma2/blob/master/lib/python/Plugins/SystemPlugins/CableScan/plugin.py

**Comment by retrofan1979** [comment]
And is it possible to add spacers between unused channel numbers instead of markers? With markers I see '-' in the channellist with unused channels. With spacers you don't see this behaviour which I prefer personally.

**Comment by Huevos** [comment]
What image are you using?

**Comment by retrofan1979** [comment]
Hi, I'm using OpenATV and is compatible with OpenVIX?

**Comment by retrofan1979** [comment]
I also tried myself to add the inversion parameter, but on OpenATV the cablescan plugin crashes. Maybe I'm doing something wrong? I altered the original plugin.py for OpenATV which is a little bit different compared to OpenVIX version?

https://github.com/openatv/enigma2/issues/1805#issuecomment-674835963

**Comment by Huevos** [comment]
Sorry I don't understand what you are asking. You want us to a modification to cablescan plugin so you can take the IPK and use it in another distro?

**Comment by retrofan1979** [comment]
I'm using OpenATV but this issue also applies to OpenVIX. My guess is that when you fix this issue for OpenVIX I also could use the fix on OpenATV with a new compiled IPK. If I'm wrong, can you please fix the issue for OpenATV then? I thought both images are compatible, anyway, the options in the menu looks very similar.

**Comment by Huevos** [comment]
If you are using OpenATV enigma2 and something needs fixing you should post about it on ATV forum or repo.

**Comment by retrofan1979** [comment]
Yes I know, I alread did, but the same goes for OpenVIX. If you fix it, I also can try OpenVIX.

**Comment by retrofan1979** [comment]
Please investigate this issue, so I can try OpenVIX as well.
